examples/lifecycle/client.ts

```diff
@@ -297,53 +316,59 @@ export class LifeCycleAppClient {
     return this.appClient.deploy({
       ...params,
-      createArgs: Array.isArray(params.createArgs) ? mapBySignature(...params.createArgs as [any, any, any]): params.createArgs,
-      createOnCompleteAction: Array.isArray(params.createArgs) ? params.createArgs[2]?.onCompleteAction : params.createArgs?.onCompleteAction,
-      updateArgs: Array.isArray(params.updateArgs) ? mapBySignature(...params.updateArgs as [any, any, any]): params.updateArgs,
+      createOnCompleteAction: params.createArgs?.onCompleteAction,
     })
   }
 
   /**
-   * Creates a new instance of the LifeCycleApp smart contract using a bare call.
-   * @param args The arguments for the bare call
-   * @returns The create result
-   */
-  public create(args: BareCallArgs & AppClientCallCoreParams & AppClientCompilationParams & CoreAppCallArgs & (OnCompleteNoOp | OnCompleteOptIn)): Promise<AppCallTransactionResultOfType<undefined>>;
-  /**
-   * Creates a new instance of the LifeCycleApp smart contract using the create(string)string ABI method.
-   * @param method The ABI method to use
-   * @param args The arguments for the contract call
-   * @param params Any additional parameters for the call
-   * @returns The create result
-   */
-  public create(method: 'create(string)string', args: MethodArgs<'create(string)string'>, params?: AppClientCallCoreParams & AppClientCompilationParams  & (OnCompleteNoOp)): Promise<AppCallTransactionResultOfType<MethodReturn<'create(string)string'>>>;
-  /**
-   * Creates a new instance of the LifeCycleApp smart contract using the create(string,uint32)void ABI method.
-   * @param method The ABI method to use
-   * @param args The arguments for the contract call
-   * @param params Any additional parameters for the call
-   * @returns The create result
+   * Gets available create methods
    */
-  public create(method: 'create(string,uint32)void', args: MethodArgs<'create(string,uint32)void'>, params?: AppClientCallCoreParams & AppClientCompilationParams  & (OnCompleteNoOp)): Promise<AppCallTransactionResultOfType<MethodReturn<'create(string,uint32)void'>>>;
-  public create(...args: any[]): Promise<AppCallTransactionResultOfType<unknown>> {
-    if(typeof args[0] !== 'string') {
-      return this.appClient.create({...args[0], })
-    } else {
-      return this.mapReturnValue(this.appClient.create({ ...mapBySignature(args[0] as any, args[1], args[2]), }))
+  public get create() {
+    const $this = this
+    return {
+      /**
+       * Creates a new instance of the LifeCycleApp smart contract using a bare call.
+       * @param args The arguments for the bare call
+       * @returns The create result
+       */
+      bare(args: BareCallArgs & AppClientCallCoreParams & AppClientCompilationParams & CoreAppCallArgs & (OnCompleteNoOp | OnCompleteOptIn) = {}): Promise<AppCallTransactionResultOfType<undefined | ABIReturn>> {
+        return $this.appClient.create({...args, })
```

Comment:
I don't think any of the object spreads here are required anymore as we don't merge objects - should we remove them?